website/src/docs-theme/components/AppHeader.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <chakra.header
+    ref="headerRef"
+    :shadow="y > height ? 'sm' : undefined"
+    transition="box-shadow 0.2s, background-color 0.2s"
+    pos="sticky"
+    top="0"
+    zIndex="3"
+    :bg="bg"
+    left="0"
+    right="0"
+    width="full"
+  >
+    <chakra.div height="4.5rem" mx="auto" maxW="1200px">
+      <!-- content -->
+
+      <CFlex w="100%" h="100%" px="6" align="center" justify="space-between">
+        <CFlex align="center">
+          <router-link to="/">
+            <chakra.a display="block" aria-label="Chakra UI, Back to homepage">
+              <Logo :display="{ base: 'none', md: 'block' }" />
+              <CBox min-w="3rem" :display="{ base: 'block', md: 'none' }">
+                <LogoIcon />
+              </CBox>
+            </chakra.a>
+          </router-link>
+        </CFlex>
+
+        <!-- nav -->
+        <CFlex
+          justify="flex-end"
+          w="100%"
+          maxW="824px"
+          align="center"
+          color="gray.400"
+        >
+          <chakra.div>Search</chakra.div>
+          <chakra.div>Switcher</chakra.div>
+          <CFlex
+            todo="HStack"
```

Comment:
what does the ```todo``` prop do? 😊

Reply:
again just ignore it, it was a todo for myself :D